MyLogger.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import os
+import datetime
+
+
+class MyLogger:
+    _logger = None
+
+    def __new__(cls, *args, **kwargs):
+        dirname = "./logs"
```

Comment:
perhaps should add smth like `if _logger is not null: return _logger`? For singleton pattern. Would that work?

Reply:
Done!